src/iotjs_binding.cpp

```diff
@@ -139,142 +115,126 @@ JObject::~JObject() {
 }
 
 
-static JRawValueType jnull_val = JVal::Null();
-static JObject jnull(&jnull_val);
-JObject& JObject::Null() {
-  return jnull;
+JObject JObject::Null() {
+  return JObject(JVal::Null(), false);
```

Comment:
@akosthekiss, thanks. Lazy initialization works fine for me. I'll update the patch.